packages/end-user-strings/src/quick-fix.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Quick fix commands strings
+ */
+
+import { QuickFixCommands } from "../api";
+
+export const quickFixCommands: QuickFixCommands = {
+  STABLE_ID_ERROR: {
```

Comment:
could this be made generic? or do only quickFix scenario has both a `command` and a `title`?